Src/AutoFixture.xUnit.net2/FrozenAttribute.cs

```diff
@@ -11,22 +13,63 @@ namespace Ploeh.AutoFixture.Xunit2
     [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = false)]
     public sealed class FrozenAttribute : CustomizeAttribute
     {
+        private readonly Matching by;
+        private IRequestSpecification matcher;
```

Comment:
Could we please get rid of this [Temporary Field](http://blog.ploeh.dk/2015/09/18/temporary-field-code-smell)?